### PR TITLE
IHP plugins Platform PR

### PR DESCRIPTION
**Pull request by saina-ramyar**
# PR Details
## Description

IHP Strategic and Control plugins are added and other plugins updated to work with them

## Related Issue
https://github.com/usdot-fhwa-stol/carma-platform/issues/1826

## Motivation and Context
Needed for IHP2 functionality

## How Has This Been Tested?
Tested on Black Pacifica, Blue Lexus and White Fusion.

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by msmcconnell — CHANGES_REQUESTED** [review]

File: arbitrator/config/arbitrator_params.yaml [hunk 1]
```diff
@@ -26,4 +26,5 @@ use_fixed_costs: true
 # Map: The priorities associated with each plugin during the planning 
 # process, values will be normalized at runtime and inverted into costs
 # Unit: N/a
-plugin_priorities: {LCIStrategicPlugin: 5.0, SCIStrategicPlugin: 4.0, RouteFollowingPlugin: 2.0, WorkZonePlugin: 4.0, PlatooningStrategicPlugin: 1.0}
+plugin_priorities: {RouteFollowingPlugin: 1.0, WorkZonePlugin: 1.0, PlatooningStrategicIHPPlugin: 4.0}
+# plugin_priorities: {LCIStrategicPlugin: 5.0, SCIStrategicPlugin: 4.0, RouteFollowingPlugin: 2.0, WorkZonePlugin: 4.0, PlatooningStrategicPlugin: 1.0}
```
[the remark this answers is not in the document]
Comment (saina-ramyar) [reply arbitrator/config/arbitrator_params.yaml]: done, also added the original platooning plugin

File: carma/launch/environment.launch.py [hunk 2]
```diff
@@ -223,97 +238,98 @@ def generate_launch_description():
...
-                    remappings=[
-                        ("georeference", [ EnvironmentVariable('CARMA_LOCZ_NS', default_value=''), "/map_param_loader/georeference" ] ),
-                        ("geofence", [ EnvironmentVariable('CARMA_MSG_NS', default_value=''), "/incoming_geofence_control" ] ),
-                        ("incoming_mobility_operation", [ EnvironmentVariable('CARMA_MSG_NS', default_value=''), "/incoming_mobility_operation" ] ),
-                        ("incoming_spat", [ EnvironmentVariable('CARMA_MSG_NS', default_value=''), "/incoming_spat" ] ),
-                        ("route", [ EnvironmentVariable('CARMA_GUIDE_NS', default_value=''), "/route" ] )
-                    ]
+                parameters=[ carma_wm_ctrl_param_file, vehicle_config_param_file, vehicle_characteristics_param_file ]
             ),
+            # ComposableNode(
```
[the remark this answers is not in the document]
Comment (saina-ramyar) [reply carma/launch/environment.launch.py]: done

File: carma/launch/guidance.launch [hunk 3]
```diff
@@ -223,7 +234,7 @@
   <include file="$(find stop_controlled_intersection_tactical_plugin)/launch/stop_controlled_intersection_tactical_plugin.launch" />
 
   <!-- Light Controlled Intersection Tactical Plugin -->
-  <include file="$(find light_controlled_intersection_tactical_plugin)/launch/light_controlled_intersection_tactical_plugin.launch" />
+  <!-- <include file="$(find light_controlled_intersection_tactical_plugin)/launch/light_controlled_intersection_tactical_plugin.launch" /> -->
```
[the remark this answers is not in the document]
Comment (saina-ramyar) [reply carma/launch/guidance.launch]: I remember when we first merged with develop, this plugin was throwing build errors, so we just excluded it from the build and commented the launch:)

File: docker/install.sh [hunk 4]
```diff
@@ -47,7 +47,7 @@ if [[ ! -z "$ROS1_PACKAGES$ROS2_PACKAGES" ]]; then
     fi
 else
     echo "Building all ROS1 CARMA Components"
-    colcon build --install-base /opt/carma/install --cmake-args -DCMAKE_BUILD_TYPE=Release
+    colcon build --install-base /opt/carma/install --packages-skip light_controlled_intersection_tactical_plugin --cmake-args -DCMAKE_BUILD_TYPE=Release
```
[the remark this answers is not in the document]
Comment (saina-ramyar) [reply docker/install.sh]: done

File: inlanecruising_plugin/config/parameters.yaml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # The length of the trajectory in time domain, in seconds
-trajectory_time_length: 6.0 # Trajectory length in seconds
+trajectory_time_length: 10.0 # Trajectory length in seconds
```
[the remark this answers is not in the document]
Comment (saina-ramyar) [reply inlanecruising_plugin/config/parameters.yaml]: done

File: localization_manager/config/parameters.yaml [hunk 6]
```diff
@@ -39,4 +39,4 @@ sequential_timesteps_until_gps_operation: 5
 # 2 - AUTO select between NDT and GNSS with GNSS timeout
 # 3 - AUTO select between NDT and GNSS without GNSS timeout
 # 4 - GNSS only with NDT initialization
-localization_mode: 4
+localization_mode: 1
```
[the remark this answers is not in the document]
Comment (saina-ramyar) [reply localization_manager/config/parameters.yaml]: done

**Pull request merged**